install.sh

```diff
@@ -10,6 +10,8 @@
 # Script options:
 # --build-icu              Builds ICU libraries.
 # --build-boost            Builds Boost libraries.
+# --build-qrencode         Builds QREncode libraries.
+# --build-libpng           Builds LIBPNG libraries.
```

Comment:
For consistency and brevity I'd like to drop "lib" from `--build-libpng` and `WITH_LIBPNG`.